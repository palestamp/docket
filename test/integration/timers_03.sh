rm ~/.timer
set -o verbose
docket timer new work --abstract
docket timer new work_child --parent work
docket timer start work
