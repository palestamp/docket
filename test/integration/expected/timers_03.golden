docket timer new work --abstract
docket timer new work_child --parent work
docket timer start work
error: Abstract timer can not be manually started
